cypress.json

```diff
@@ -1,10 +1,11 @@
 {
   "baseUrl": "https://qa.nrs-ghtr.org.uk/",
-  "accessibility": false,
+  "accessibility": true,
   "reporter": "spec",
   "reporterEnabled": "junit,spec",
   "reporterOptions": {
     "mochaFile": "results/test-output-[hash].xml",
     "toConsole": true
-  }
+  },
+  "chromeWebSecurity": false
```

Comment:
What is this option is disabling?

Reply:
Originally it was to access cross-origin iframes that are embedded in the application but we don't need this so removing.